ale_linters/less/lessc.vim

```diff
@@ -0,0 +1,36 @@
+" Author: zanona <https://github.com/zanona>, w0rp <devw0rp@gmail.com>
+" Description: This file adds support for checking Less code with lessc.
+
+call ale#Set('less_lessc_options', '')
+
+function! ale_linters#less#lessc#GetCommand(buffer) abort
+    return 'lessc'
+          \ . ' --no-color --lint '
+          \ . ale#Var(a:buffer, 'less_lessc_options')
+          \ . ' %t'
```

Comment:
Align the line line continuations a little differently. Look through the rest of the codebase for examples.